src/main/java/org/jabref/gui/preview/CopyCitationAction.java

```diff
@@ -87,7 +88,12 @@ private List<String> generateCitations() throws IOException {
         }
 
         if (styleSource != null) {
-            return CitationStyleGenerator.generateCitations(selectedEntries, styleSource, outputFormat, stateManager.getActiveDatabase().get(), Globals.entryTypesManager);
+            return CitationStyleGenerator.generateCitations(
+                    selectedEntries,
+                    styleSource,
+                    outputFormat,
+                    stateManager.getActiveDatabase().get(),
+                    Injector.instantiateModelOrService(BibEntryTypesManager.class));
```

Comment:
Can't the `Injector` be used inside `CitationStyleGenerator`?

Reply:
Yes, could be, but using a proper DI framework is just the next iteration step.